tests/pytests/functional/pillar/test_makostack.py

```diff
@@ -0,0 +1,134 @@
+import pytest
+import salt.pillar
+
+
+@pytest.fixture()
+def makostack_basic_config(tmp_path):
+    mako_config = tmp_path / "mako" / "mako.cfg"
```

Comment:
Would you want to do a `with tmp_path` here so that it gets cleaned up?

Reply:
tmp_path is (or should be) automatically cleaned up by the fixture. If it's not then that's broken across the board :upside_down_face: